app/pages/learn/protocols/uniswap-v3.mdx

```diff
@@ -147,7 +152,7 @@ Collect accumulated fees from a specific NFT position.
   | `params.amount0Max` | -                 |
   | `params.amount1Max` | -                 |
 
-### If WETH is involved
+### If WETH is involved, the following functions are also whitelisted:
```

Comment:
The longer title is a bit ugly to look at in the nav:

<img width="261" alt="image" src="https://github.com/karpatkey/defi-kit/assets/524089/bd2abef3-4493-4fe2-9e04-70860ec3736c">

I'd vote for the short form, but you can make the call 👍